### DISCUSSION TITLE
Nimib speaking hours 🎪 - launch and November 27th edition

### DISCUSSION THREAD

**Discussion by pietroppeter** [pietroppeter/nimib#143] [Nimib speaking hours]
me and @HugoGranstrom started meeting regularly while preparing slides for NimConf 2022 and we will keep doing it and we thought we might as well open it to anyone who wants to jump in and say hi or ask stuff.
we will be advertising the time for the next meeting here, for the moment we just know it should be second half of November. Ah, it will be on weekends!

**Comment by Vindaar** [comment]
Ohh, nice! I'll join if nothing else comes up at the time!

**Reply by pietroppeter** [reply]
that would be great!

**Reply by HugoGranstrom** [reply]
Looking forward to seeing you there! :D

**Comment by pietroppeter** [comment]
Hey, we will have to start thinking about when we want to meet for the November speaking hours! Next weekend on Sunday (Nov 20)?

**Reply by HugoGranstrom** [reply]
Sunday should work for me 👍 what time should we say? 12:00?

**Reply by pietroppeter** [reply]
mmh, I would keep it a bit farther from lunch time (although I guess for you would be fika, right?). I could go with before 12 or after 14 (not that I plan to eat for 2 hours...). @Vindaar do you have preferences? (in case you think you might be able to join...)

**Reply by pietroppeter** [reply]
as for tech to use for the call, I liked that jitsi that was used during nim dev meetings (https://meet.jit.si), although I am pretty ignorant about potential alternatives...

**Reply by HugoGranstrom** [reply]
Haha, I have fika at 14 usually so 12-15 seems like a no-go time :rofl: Both in the morning and afternoon works for me, although I would prefer the morning. 

I've never used Jitsi before, but it seems like a good options. Free, no installation, no accounts :+1:

**Reply by pietroppeter** [reply]
so I guess right now the first option on the table is 11-12? let's see if @Vindaar (or anyone else! don't be shy dear lurkers!) has a preference...

**Reply by HugoGranstrom** [reply]
Yes, that sounds like a good plan A :+1:  And we should probably mention we are talking times in Central European Time (CET) :rofl:

**Reply by Vindaar** [reply]
Next weekend for me it will depend. I'll be away from home. I _might_ be able to join if it's around 11-12 or so, but I will only be able to tell closer towards the weekend (at the latest on Saturday I think).

**Reply by pietroppeter** [reply]
Would it be better for you the weekend after (26-27)? Honestly to me the next or the following are the same

**Reply by HugoGranstrom** [reply]
Ping @Vindaar

**Reply by Vindaar** [reply]
Sorry, I forgot to update you!
Next weekend would definitely be better for me. Otherwise I will try to join tomorrow, but I worry I won't be able to (won't be home at the time and while I do have time in principle, I'm not sure if the place I'm at provides a quiet place for me to join a meeting :/).

**Reply by pietroppeter** [reply]
For me is fine to postpone to next week, what do you think Hugo? I did not prepare the jitsi meet yet :) (edited: it was not what I meant earlier)

**Reply by HugoGranstrom** [reply]
It's fine by me to postpone it 😄 11-12 CET next sunday? I haven't looked into jitsi more, will have to try it in the coming week

**Reply by pietroppeter** [reply]
Ok then it's postponed to next week (Sunday 11-12 ok for me). Yeah, will have to look into jitsi too

**Reply by pietroppeter** [reply]
Sunday and the first nimib speaking hours is coming! should we make some kind of more visible announcement/reminder (forum, personal twitter/mastodon)? honestly being the first round I would be fine if we manage to just make it happen (have not had the chance yet to play with jitsi, by tomorrow I will) Maybe when we are ready we could pop a reminder/causal note in the chat. @Vindaar you are still in?

**Reply by pietroppeter** [reply]
or maybe just start a new thread here (I guess the responses in this thread are not that visible, a new thread with the date and time would be better).

I am also starting to think about agenda. For sure we could review recent changes in nimib (the recent js improvement by Hugo) and see what has priority. I did start some work about the nimib based static site generator (still private) and would gladly talk about that but then I was derailed by the joy of p5nim (which is anyway a website built with nimib and where I already know I took a wrong turn with respect to SSG)... also we could talk about upcoming advent of code... I know if @Vindaar is there (but also if he isn't) we could talk about the toHtml added to datamancer and what is missing to have it work for the penguins example in nimib (IIRC it is very little work).

**Reply by Vindaar** [reply]
Yeah, I'll be around. I'm also fine if it's a small round for the first time, but don't mind if you post about it elsewhere either. Topic wise that sounds good!

Btw, I know jitsi pretty well.

**Reply by HugoGranstrom** [reply]
I agree, we have a lot of things to sort out about these discussion hours so no forum thread this time, but we could drop this on other places like Mastodon where we have a smaller audience. 

> or maybe just start a new thread here (I guess the responses in this thread are not that visible, a new thread with the date and time would be better).

And now I finally have activated my notifications for discussions 🤣 Didn't see your p5 thread until now... Discussion forums here on Github doesn't have the greatest visibility sadly, I think we will have to remind people who are interested to actively go in and activate "Discussions" notifications for this repo. 

Agenda-wise that sounds good :+1: It should keep us occupied for the hour. Would love to hear more about the HydeAndSeek SSG and your p5 adventures :D

**Reply by pietroppeter** [reply]
>  I finally have activated my notifications for discussions

ah, you have to activate explicitly? yeah this discussion forum is not very visible/discoverable (we probably could improve readme about that). @Vindaar how did you find out about it? I fear me and Hugo are the only ones that know it exists :D

I'd say then that no further messaging for this first round. @Vindaar I am guessing that starting from zero, to set up jitsi and plan a call it should not be too difficult, right?

**Reply by Vindaar** [reply]
I think you automatically get notified, if you say "watch all activity" of the repository. But I could be mistaken.

> I am guessing that starting from zero, to set up jitsi and plan a call it should not be too difficult, right?

It is as simple as going to:

https://meet.jit.si/YourFavoriteUrlYouWishToUseToTalkAboutNimib

Visiting any url will start a meeting under that URL. Once you enter you can place a password if desired.

**Reply by pietroppeter** [reply]
well, that's indeed easy. Then I guess it will be: https://meet.jit.si/NimibSpeakingHours

**Reply by pietroppeter** [reply]
(I just went in the meeting to test it if you want to say a quick hi!)

**Reply by HugoGranstrom** [reply]
> ah, you have to activate explicitly?

I always use the "custom notifications" option for repos that aren't my own and usually only ticks the `Releases` and perhaps `PRs` options. So if people in general are behaving similarly they might not choose the "Discussions" option as well. 

> I think you automatically get notified, if you say "watch all activity" of the repository. But I could be mistaken.

Yes correct, my options are probably from before I joined nimib so I didn't want all notifications back then :P

> yeah this discussion forum is not very visible/discoverable (we probably could improve readme about that)

Yes adding it to the readme is a good first step. Just the fact that it doesn't show a number like for issues and PRs makes it look like there aren't any discussions.

**Comment by pietroppeter** [comment]
we are in @Vindaar!

**Reply by pietroppeter** [reply]
https://meet.jit.si/NimibSpeakingHours
(anyone else in welcome too)
